pkg/trace/api/api.go

```diff
@@ -128,6 +128,7 @@ func (r *HTTPReceiver) Start() {
 	mux.HandleFunc("/v0.3/services", r.handleWithVersion(v03, r.handleServices))
 	mux.HandleFunc("/v0.4/traces", r.handleWithVersion(v04, r.handleTraces))
 	mux.HandleFunc("/v0.4/services", r.handleWithVersion(v04, r.handleServices))
+	mux.Handle("/profiling/v1/input", r.profileProxyHandler())
```

Comment:
Yeah I like this option a lot:
* Namespaces it to `profiling` so we don't get mixed with trace endpoints.
* Adds versioning.
* Matches semantics in serverless intake.
* Hides the proxying behaviour as an implementation detail.